pkg/tests/ossm/prometheus_scoped_scraping_test.go

```diff
@@ -191,7 +191,7 @@ func checkPermissionError(t test.TestHelper) {
 }
 
 func checkConfigurationReloadingTriggered(t test.TestHelper, start time.Time) {
-	retry.UntilSuccess(t, func(t test.TestHelper) {
+	retry.UntilSuccessWithOptions(t, retry.Options().DelayBetweenAttempts(1*time.Minute), func(t test.TestHelper) {
```

Comment:
The default number of retries is still 60, right? So this test will take 1h to fail?

I suggest you shorten this. The test really shouldn't run for more than 3-5 minutes max.

Reply:
Changed to wait one minute then try for another minute.